rust/connlib/snownet/src/node.rs

```diff
@@ -1530,7 +1530,7 @@ where
                     self.state.add_possible_socket(source);
                 }
                 IceAgentEvent::IceConnectionStateChange(IceConnectionState::Disconnected) => {
-                    tracing::info!("Connection failed (ICE timeout)");
+                    tracing::error!("Connection failed (ICE timeout)");
```

Comment:
This also happens if a client just shuts down (you close your laptop). Do we really want to log these as errors then? It will be quite common on the gateway, not so much on the client.

Reply:
👍

Reply:
To fix this, it would be nice to have a control protocol between client and gateway, then we could terminate a connection gracefully instead of having it time-out.